src/transformStateWithClones.js

```diff
@@ -8,6 +8,29 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
+  let newArray = { ...state };
```

Comment:
Приділяй назвам змінних більше стану

```suggestion
function transformStateWithClones(originState, actions) {
  // write code here
  const result = [];
  let state = { ...originState };
```